tests/nativeapp/test_commands.py

```diff
@@ -2,13 +2,11 @@
 
 import pytest
 
-from tests.testing_utils.fixtures import *
-
 PROJECT_PATH = "demo_na_project"
 
 
 @pytest.mark.parametrize(
-    "id,init_args",
+    "id_,init_args",
```

Comment:
Are `id` changes to avoid conflicting with `id()` method in Python?

Reply:
Correct